src/main/java/racingcar/domain/CarMovement.java

```diff
@@ -0,0 +1,25 @@
+package racingcar.domain;
+
+public class CarMovement {
+
+    private final String OUT_OF_MOVEMENT_RANGE_EXCEPTION = "범위 밖의 움직임 값입니다.";
+    private final Integer value;
+
+    public CarMovement(Integer value){
+        this.validate(value);
+        this.value = value;
+    }
+
+    private void validate(Integer value){
+        if(value >= 10 || value < 0){
+            throw new IllegalArgumentException(OUT_OF_MOVEMENT_RANGE_EXCEPTION);
+        }
+    }
+
+    public boolean isMove(){
+        if(this.value >= 4){
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return this.value >= 4;
```
더 간결하게도 가능할 것 같아요 ㅎㅎ

Reply:
감사합니다!! 이런 메서드를 짤 때 머리 속에 먼저 떠오르는 대로 손이 먼저 나가네요.. 항상 더 좋은 방법을 생각해보는 습관을 들여야 할 것 같습니다!!